src/main/java/org/broadinstitute/consent/http/service/SummaryService.java

```diff
@@ -153,7 +153,7 @@ public List<ConsentSummaryDetail> describeConsentSummaryDetail() {
             Stream.of(ElectionStatus.CLOSED.getValue(), ElectionStatus.CANCELED.getValue())
                 .map(String::toLowerCase)
                 .collect(Collectors.toList());
-        List<Election> reviewedElections = electionDAO.findElectionsWithFinalVoteByTypeAndStatus(ElectionType.TRANSLATE_DUL.getValue(), statuses).stream().filter(e -> Objects.nonNull(e.getFinalVote())).distinct().collect(Collectors.toUnmodifiableList());
+        List<Election> reviewedElections = electionDAO.findElectionsWithFinalVoteByTypeAndStatus(ElectionType.TRANSLATE_DUL.getValue(), statuses).stream().filter(e -> Objects.nonNull(e.getFinalVote())).distinct().collect(Collectors.toList());
```

Comment:
nit: Changes here seem unrelated to the intent of the PR, I would revert these. It is true, that in the past, I've had issues with `toUnmodifiableList`, that's not the case here.